src/main/java/net/fabricmc/loader/gui/FabricMainWindow.java

```diff
@@ -92,7 +93,13 @@ private static void open0(FabricStatusTree tree, boolean shouldWait) throws Exce
 	private static void createUi(CountDownLatch onCloseLatch, FabricStatusTree tree) {
 		JFrame window = new JFrame();
 		window.setVisible(false);
-		window.setTitle("Fabric Loader");
+
+		window.setTitle("Fabric Loader " + FabricLoader.getInstance()
+			.getModContainer("fabricloader")
```

Comment:
Has this been tested? Im not sure this will be populated if mod resolution fails.

I would at the very least handle fabricloader not being here, and make a it a var as its kinda gross being put here with the 5 lines of calls.

Reply:
FIxed in 85fbf6a584090562b82899f7017a774a3b28a254